### PR TITLE
Finish gradient for tf.pow

### PR DESCRIPTION
**Pull request by jgartman**
This PR finishes the implementation of the gradient for `tf.pow`. https://github.com/tensorflow/tfjs/issues/108

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/954)

### CODE REVIEW

**Comment by dsmilkov** [comment]
:lgtm_strong: Thank you! Really nice work on reusing y=base^exp for the backprop.

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-core/954#-:-LAGH1Dj1VuWio3l7e7l:bxg283a)*

**Comment by dsmilkov** [comment]
There is some conflicting file. Let me know when you resolve and I'll merge this in.

**Comment by jgartman** [comment]
@dsmilkov I think it should be good to go.
